dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1464,12 +1476,75 @@
         }
       ]
     },
+    "EntitiesLROTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for an Entity Recognition LRO task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EntitiesTaskParameters"
+        },
+        {
+          "$ref": "#/definitions/EntitiesUDEParameters"
+        }
+      ]
+    },
+    "EntitiesUDEParameters": {
+      "type": "object",
+      "description": "Parameters object for a text analysis task using user defined entities.",
+      "properties": {
+        "userDefinedEntities": {
+          "$ref": "#/definitions/UserDefinedEntities"
+        }
+      }
+    },
+    "UserDefinedEntities":{
+      "type": "object",
+      "description": "The container object for user defined entities parameter definition.",
+      "properties": {
+        "entities": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "$ref": "#/definitions/UserDefinedEntityDefinition"
+          }
+        },
+        "samples": {
+          "type": "array",
+          "description": "List of sample sentences to provide contextual examples to the model to better learn about the entity.",
+          "maxItems": 5,
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "entities",
+        "samples"
+      ]
+    },
+    "UserDefinedEntityDefinition":{
+      "type": "object",
+      "description": "The user defined entity definition.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The name of the user defined entity."
+        },
+        "hint": {
+          "type": "string",
+          "description": "A description or hint to provide to the model to better understand the type of the entity."
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
     "EntitiesLROTask": {
       "type": "object",
       "description": "An object representing the task definition for an Entities Recognition task.",
       "properties": {
         "parameters": {
-          "$ref": "#/definitions/EntitiesTaskParameters"
+          "$ref": "#/definitions/EntitiesLROTaskParameters"
```

Comment:
Presumably this isn't breaking the REST API because the new model includes the old model, but if this already GA'd it will be an SDK-breaking change since the SDK supports multiple versions.

Reply:
Shouldn't be any breaking change. Only new parameters. I might have messed up the swagger :-) Fixing it now